omnibus/lib/ostools.rb

```diff
@@ -67,3 +67,7 @@ def with_python_runtime?(runtime)
     python_runtimes = ENV['PY_RUNTIMES'].nil? ? ['3'] : ENV['PY_RUNTIMES'].split(',')
     return python_runtimes.include? runtime
 end
+
+def fips_mode?()
+    return ENV['FIPS_MODE'] == 'true' && (linux_target? || windows_target?)
+end
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏Can you add a newline?